### PR TITLE
Fixes #940 - updated Twitter link

### LINKED ISSUE #940: Change Twitter link in footer
We've swapped the @Webmaker and @MozTeach Twitter handles to prepare for the Webmaker app launch. 

Can we change the Twitter hyperlink in the teach.mozilla.org footer to link to twitter.com/MozTeach instead of twitter.com/Webmaker? 

<Screen Shot 2015-05-22 at 5.16.44 PM.png>

We also need to update the link on the About page (bottom of page, 1st CTA)

cc @KevZawacki @toolness @mmmavis

### PR DESCRIPTION
**Pull request by mmmavis**
This fixes #940.

### CODE REVIEW

File: pages/clubs-toolkit.jsx [hunk 1]
```diff
@@ -79,7 +81,7 @@ var ClubsToolkit = React.createClass({
                   Read what others have done and get your questions answered.
                 </ToolkitListItem>
                 <ToolkitListItem header="Twitter.">
-                  Our community loves to tweet! Share ideas and send pictures to <a href="https://twitter.com/webmakerhttps://twitter.com/search?q=%23teachtheweb">@Webmaker</a> and use the tag <a href="https://twitter.com/search?q=%23teachtheweb">#TeachTheWeb</a>
```
Comment (mmmavis) [inline pages/clubs-toolkit.jsx]: spotted a bug in url here :grin:

**Comment by mmmavis** [comment]
@toolness review for me plz? :smile:

File: pages/clubs-toolkit.jsx [hunk 2]
```diff
@@ -184,7 +186,7 @@ var ClubsToolkit = React.createClass({
               <p>Tell the world what you and your club are doing! Here are ways to show off the hard work that’s put into teaching and learning the web.</p>
               <ToolkitList>
                 <ToolkitListItem header="Social Media.">
-                  Share your club activities online. You might even consider creating a new hashtag just for your club so the learners can keep track of all that happens in one place. Write a few sample tweets and posts that your mentors and learners can share on their networks as well. Be sure to share them with us too using <a href="https://twitter.com/search?q=%23teachtheweb">#teachtheweb</a> and <a href="https://twitter.com/webmaker">@Webmaker</a>.
+                  Share your club activities online. You might even consider creating a new hashtag just for your club so the learners can keep track of all that happens in one place. Write a few sample tweets and posts that your mentors and learners can share on their networks as well. Be sure to share them with us too using <a href="https://twitter.com/search?q=%23teachtheweb">#teachtheweb</a> and <a href={config.TWITTER_LINK}>@Webmaker</a>.
```
Comment (toolness) [inline pages/clubs-toolkit.jsx]: Hmm it looks like the actual text of the link still says `@Webmaker`, maybe make a `config.TWITTER_NAME` too?

File: pages/clubs-toolkit.jsx (hunk 1 above)
Comment (toolness) [inline pages/clubs-toolkit.jsx]: ah nice catch!

File: lib/config.js [hunk 3]
```diff
@@ -25,4 +27,5 @@ exports.GENERATING_STATIC_SITE = GENERATING_STATIC_SITE;
 exports.ENABLE_PUSHSTATE = ENABLE_PUSHSTATE;
 exports.IN_TEST_SUITE = (typeof describe == 'function');
 exports.DEV_SERVER_PORT = DEV_SERVER_PORT;
+exports.TWITTER_LINK = TWITTER_LINK;
```
Comment (toolness) [inline lib/config.js]: Cool. Just so you know, you're also welcome to just write `exports.TWITTER_LINK = "https://twitter.com/MozTeach"` if you want, it's up to you. I think I just started defining vars at the top so I could reuse them in the definition of other consts without having to prefix them with `exports.` or something.

Comment (mmmavis) [inline lib/config.js]: agree.  this is what happened when i multitasked :)

**Comment by toolness** [comment]
Ah stupid linting failed!

```
Message:
    Invalid quote mark found at config.js :
     5 |                        window.history.replaceState);
     6 |var DEV_SERVER_PORT = 8008;
     7 |var TWITTER_LINK = "https://twitter.com/MozTeach";
---------------------------^
     8 |var ORIGIN;
     9 |
```

Can you fix that and the Twitter username issue I mentioned [above](https://github.com/mozilla/teach.webmaker.org/pull/944#discussion_r31079958)? After that, if Travis is happy, feel free to merge.

**Comment by mmmavis** [comment]
okie dokie now waiting for Travis to finish.  Thanks @toolness

**Comment by mmmavis** [comment]
oops let me fix the handle thingy

**Pull request merged**